slsim/lsst_science_pipeline.py

```diff
@@ -762,6 +765,214 @@ def dp0_time_series_images_data(butler, center_coord, radius="0.1", band="i", si
     return table_data
 
 
+def opsim_time_series_images_data(
+    ra_list,
+    dec_list,
+    obs_strategy,
+    MJD_min=60000,
```

Comment:
Hi @Nikki1510, It might be good to mention why you chose these default values (if there is any reason). Also, supernovae lightcurves or quasar lightcurves are in days. So, using MJD time to get magnitude from the interpolated lightcurve might not work. I think we need to change them in days (relative difference).